apps/elewa-website/src/app/app.component.ts

```diff
@@ -7,4 +7,12 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'elewa-website';
+
+  /*  Button function; when button is clicked, an empty/void event is sent  */
+  
+  Click(): void {
+    /* an empty output. */
+  
```

Comment:
This shouldn't be here

Reply:
Am suppose to create new lib pages/home and add the click event there, or where am i going wrong?
@LemmyMwaura